src/drawer/styles.scss

```diff
@@ -3,7 +3,6 @@
  SPDX-License-Identifier: Apache-2.0
 */
 
-/* stylelint-disable @cloudscape-design/no-implicit-descendant, selector-max-type */
```

Comment:
Instead of disabling for the whole file, confined the exception only for relevant styles